client/me/account-close/closed.jsx

```diff
@@ -76,27 +76,14 @@ function AccountDeletedPage() {
 					}
 				/>
 				<div className="account-deleted__buttons">
-					<Button variant="secondary" onClick={ onCancelClick }>
+					{ config.isEnabled( 'me/account-restore' ) && restoreToken && (
```

Comment:
Do we still need to use the `me/account-restore` gate since the feature was launched?

Reply:
Yup I was planning to remove it later, but let's do that now!